test/e2e-tests-standing-order.sh

```diff
@@ -38,9 +66,67 @@ step_with_retry "Make sure standing order is still traded" \
 step "Update standing order" \
 "npx truffle exec scripts/standing_order.js 0 1 2 1 2"
 
+step_with_retry "Check graph standing order batch has been updated" \
+"source ../test/utils.sh && query_graphql \
+    \"query { \
+        standingSellOrderBatches(where: { \
+          accountId: 0, \
+          batchIndex: 1, \
+          validFromAuctionId: 2 \
+        }) { \
+        orders { sellAmount } \
+      } \
+    }\" | grep 2000000000000000000"
+
+step_with_retry "Check graph standing order has been updated" \
+"source ../test/utils.sh && query_graphql \
+    \"query { \
+        sellOrders (where: { \
+          accountId: 0, \
+          auctionId: null, \
+          slotIndex: null, \
+          buyToken: 2, \
+          sellToken: 1, \
+          buyAmount: \"1000000000000000000\", \
+          sellAmount: \"2000000000000000000\" \
+        }) { \
+          buyAmount \
+        } \
+    }\" | grep 1000000000000000000"
+
 step "Cancel standing order in same batch (make sure only cancel gets processed)" \
 "npx truffle exec scripts/standing_order.js 0 0 0 0 0"
 
+# TODO: Review after the PR that discard orders with sellVolume 0 
```

Comment:
Can you create an issue for Todos you intend to merge and reference the issue in the comment?